app/src/components/market/sections/market_profile/market_status/open.tsx

```diff
@@ -138,13 +188,34 @@ const Wrapper = (props: Props) => {
     </Button>
   )
 
+  const finalizeButtons = (
+    <MarketBottomFinalizeNavGroupWrapper>
+      <Button
+        buttonType={ButtonType.secondaryLine}
+        onClick={() => {
+          window.open(`${realitioBaseUrl}/app/#!/question/${question.id}`)
```

Comment:
Is it possible to trigger the transaction to call the arbitrator directly? If not is there a more specific link that can be used?

Reply:
We def. should move this functionality to Omen! Going to create an issue for this! But this should not be part of the PR/issue.

Reply:
Created a separate issue for this: https://github.com/protofire/omen-exchange/issues/1435